### DISCUSSION TITLE
Homepage suggestion

### DISCUSSION THREAD

**Discussion by pievalentin** [lichess-org/mobile#295] [General]
I have been using the beta build for a few days and here is my feedback:

- I think having the leaderboard in the homepage is weird. I don't see the value of seeing who has 3k + elo. 
- Recent game should not be hidden so far. `Profile -> scroll quite a bit -> see latest game`. For instance, once computer analysis will be added, I think having the list of previous game on the homepage will enable us to rapidly review a previous game.
- Maybe down the line replace the twitch tab, with a study/friend/lesson tab

I am less interested in the international competitive side of chess. So it only reflects my persona.
Those could probably be improvement for the future. 

Overall great work @veloce

**Comment by veloce** [comment]
I found that the recent games were easily accessed with just one tap and a bit of scrolling... 

But of course, this is because I know where it is... I agree it makes sense to have it on the home so I'll do that. It will also push the leaderboard further down.

**Reply by ijm8710** [reply]
I agree with the OP with all his comments. I'd also like ability to hide daily puzzle but otherwise he hit the nail on the head with the importance of recent games w analysis over leaderboard leaders who we don't connect with. Yes recent games are already relatively accessible but they should be front and center for quick analysis

I can't wait for recent games to allow us to filter on variant/result/color etc and expand beyond just last 10

**Comment by pievalentin** [comment]
I tried the updated build and the latest game on the home screen make a lot more sense! 
Will keep using the app and add feedback here
Thank you, for the hard work Vincent :)

**Comment by pievalentin** [comment]
Hey @veloce I have a new feedback. I like the new app very much. One thing that could be improved is the recent game flow. For now, it is as follows:
- click on a recent game
- redirected to the "finished game screen" which suggest new game, new opponent and analysis.

I think this screen is good only when you just finished the match - not in the future when you want to review. 
IMO the flow should be like this:
- click on a recent game
- be redirected to the analysis board

The 3 current option make sense only right after a game. 
Because 
- To create a new game, the user should click the big create game in the homepage.
- it would be a weird flow to go this way to play a new game with the same opponent. I would be interested in this only right after I finished my game. 
- Only case where it could be interesting is if you have a very specific clock setting but imo it is a edge case. Most people play blitz or rapid?

I think the recent game list is just the opportunity to review my past error in the metro when connection is dodgy. 
In short, the current screen should be reserved for after the game. Coming from "recent game" I think we should redirect the user directly to the analysis board.

**Reply by veloce** [reply]
Maybe I'll improve it later, but for now it is like that because it is the only way to access the game chat once the game is finished.

**Reply by veloce** [reply]
If you want to go directly to analysis board from the game list you have the context menu (long press).

**Comment by pievalentin** [comment]
Hey @veloce, I just opened the new 0.8.9 version on IOS and the new game component removed to the top right corner. 
Now to start a game someone has to extend to the top corner versus the new game button being in the central area of the screen. Moving it away allow to show for more recent game history but I am not sure if it's very useful. 
I am curious what was the rationale. IMO the home page should focus on starting a new game and reviewing a previous game. Recent device have enough screen real estate to show both but maybe I am missing something.
Otherwise great work Vincent!

**Reply by veloce** [reply]
Fixed now.
